index.bs

```diff
@@ -1042,6 +1046,10 @@ The <dfn attribute for="XRRenderState">depthNear</dfn> attribute defines the dis
 
 Note: Typically when constructing a perspective projection matrix for rendering the developer specifies the viewing frustum and the near and far clip planes. When displaying to an [=immersive XR device=] the correct viewing frustum is determined by some combination of the optics, displays, and cameras being used. The near and far clip planes, however, may be modified by the application since the appropriate values depend on the type of content being rendered.
 
+The <dfn attribute for="XRRenderState">pasthroughFullyObscured</dfn> attribute is a hint to the {{XRSystem}} from the author to indicate that the pixels of the immersive experience completely cover the viewport.
```

Comment:
Very minor nits here: The way this is stated makes it sound a bit like a guarantee that the viewport WILL be completely covered, but we don't want to imply any validation of that. I think a minor re-wording helps disambiguate that.

```suggestion
The <dfn attribute for="XRRenderState">pasthroughFullyObscured</dfn> attribute is a hint to the {{XRSystem}} from the author to indicate that they intend to completely cover the viewport with virtual content.
```